Header header

string parent_frame_id
geometry_msgs/Pose pose

uint32 rows
uint32 cols
float32 cell_width
float32 cell_height
